manifests/agent/rubygems.pp

```diff
@@ -0,0 +1,40 @@
+class classroom::agent::rubygems (
+  Boolean $offline = false,
+) {
+  if !$offline {
+    # When online, simple package resources work fine
+    package { [ 'rspec-puppet', 'puppetlabs_spec_helper' ]:
+      ensure   => present,
+      provider => 'puppet_gem',
+    }
+
+    package { 'serverspec':
+      ensure   => present,
+      provider => 'gem',
+    }
+  } else {
+    # When offline, install gems from the /var/cache/rubygems directory
+
+    exec { 'install rspec-puppet gems':
+      command => 'gem install -l rspec-puppet-2.3.0.gem',
```

Comment:
I'm nervous about including the version number here, but this will work for now. Can you file a ticket to revisit this soon?